File: C++ Codes/Boy or Girl.cpp
#include <iostream>
#include <set>
using namespace std;
 
int main()
{
    set<char> st;
    string s;
    while(cin >> s)
    {
        for(int i = 0; i < s.length(); ++i)
            st.insert(s[i]);
        if(st.size() & 1)
            cout << "IGNORE HIM!" << '\n';
        else
            cout << "CHAT WITH HER!" << '\n';
        st.clear();
    }
    return 0;

File: Codeforces/Boy_or_girl.py
s=input()
fre={}
count=0
c=0
for i in s:
    if i in fre:
        fre[i]+=1
    else:
        fre[i]=1
#print(fre)
x=fre.values()
for i in x:
    if i==1:
        count+=1
    else:
        c+=1
f=(count+c)
if(f%2==0):
    print("CHAT WITH HER!")
else:
    print("IGNORE HIM!")

    
